src/lib/badge/badge.md

```diff
@@ -0,0 +1,33 @@
+Badges are small status descriptors for UI elements. A badge consists of a small circle, 
+typically containing a number or other short set of characters, that appears in proximity to another object.
```

Comment:
Wrap at 100 columns (here and below)

Reply:
Unclear what you are asking here, can you elaborate?